bugbug/tools/code_review.py

```diff
@@ -680,6 +717,34 @@ def collect_function_definitions(function_declarations, target_element, definiti
         )
 
 
+def gather_line_context(line_context):
+    """Reformat the line context list and remove duplicates.
+
+    Args:
+        line_context: List of lists, where each list is [line, file, function].
+
+    Returns:
+        List of lists, where each list is [gathered_line, file, function]. The
```

Comment:
We need to reflect your change from https://github.com/mozilla/bugbug/pull/4322/commits/3e5b09d5b9e7f6415a228fb155e40874bf8033be:
```suggestion
        List of tuples, where each tuple is (gathered_line, file, function). The
```